Core/Node-API/CMakeLists.txt

```diff
@@ -3,8 +3,10 @@ if(WIN32)
     set(NAPI_JAVASCRIPT_ENGINE "Chakra" CACHE STRING "JavaScript engine for Node-API")
 elseif(APPLE)
     set(NAPI_JAVASCRIPT_ENGINE "JavaScriptCore" CACHE STRING "JavaScript engine for Node-API")
-elseif(UNIX)
+elseif(ANDROID)
     set(NAPI_JAVASCRIPT_ENGINE "V8" CACHE STRING "JavaScript engine for Node-API")
+elseif(UNIX)
+    set(NAPI_JAVASCRIPT_ENGINE "JavaScriptCore" CACHE STRING "JavaScript engine for Node-API")
```

Comment:
Linux doesn't support V8? I wonder if we should be defaulting to V8?

Reply:
There is a recent V8 package that has not been tested. I have no opinion on Linux default. Linux build here is very fast (1minute). We can test both JSC and V8. Issue opened here : https://github.com/BabylonJS/JsRuntimeHost/issues/34